charts/rekor/values.yaml

```diff
@@ -120,7 +120,7 @@ server:
       existingClaim: ""
       accessModes:
         - ReadWriteOnce
-  deploymentAnnotations:
+  podAnnotations:
```

Comment:
we might add a service monitor in the future in case people we using prometheus operator. i will follow up on that